src/js/charts/LineChartIndex.js

```diff
@@ -11,8 +11,8 @@ Highcharts.setOptions( {
 } );
 
 class LineChartIndex {
-  constructor( { el, description, data, metadata, yAxisLabel } ) {
-    data = process.originations( data[0], metadata );
+  constructor( { el, description, data, metadata, source, yAxisLabel } ) {
+    data = process.originations( data[0], metadata, source );
```

Comment:
Now that this is a chart type dedicated to inquiry index and deferred denial, why can't we specify the projected value here instead of passing in another argument? It seems like the projected value could be added to the data or the metadata.